docs/examples/question_answering/squad_utils.py

```diff
@@ -275,18 +276,18 @@ def get_answerable_logits(self, F, contextual_embedding, p_mask):
             Shape (batch_size, 2)
         """
         # Shape (batch_size, sequence_length)
-        start_scores = F.np.squeeze(self.start_scores(contextual_embedding), -1)
-        start_score_weights = masked_softmax(F, start_scores, p_mask, axis=-1)
-        start_agg_feature = F.npx.batch_dot(F.np.expand_dims(start_score_weights, axis=1),
+        start_scores = np.squeeze(self.start_scores(contextual_embedding), -1)
+        start_score_weights = masked_softmax(start_scores, p_mask, axis=-1)
+        start_agg_feature = npx.batch_dot(np.expand_dims(start_score_weights, axis=1),
                                             contextual_embedding)
-        start_agg_feature = F.np.squeeze(start_agg_feature, 1)
+        start_agg_feature = np.squeeze(start_agg_feature, 1)
         cls_feature = contextual_embedding[:, 0, :]
-        answerable_scores = self.answerable_scores(F.np.concatenate([start_agg_feature,
+        answerable_scores = self.answerable_scores(np.concatenate([start_agg_feature,
                                                                      cls_feature], axis=-1))
-        answerable_logits = F.npx.log_softmax(answerable_scores, axis=-1)
+        answerable_logits = npx.log_softmax(answerable_scores, axis=-1)
         return answerable_logits
 
-    def hybrid_forward(self, F, tokens, token_types, valid_length, p_mask, start_position):
+    def forward(self, tokens, token_types, valid_length, p_mask, start_position):
```

Comment:
You may also remove the comment about `F` in the following.

Reply:
In addition, since you have fixed. Would you do the same for https://github.com/dmlc/gluon-nlp/blob/master/scripts/question_answering/squad_utils.py ?

Reply:
Sure.